codegenerator/cli/templates/static/codegen/src/Abis.res

```diff
@@ -0,0 +1 @@
+
```

Comment:
Kept the file, so it's overwritten on the codegen (locally)

Reply:
Hmmm, does that mean we should always have empty files for things we remove? I would suggest just remove it. The cli handles cleaning stale generated files if the version changes.